### PR TITLE
gsoc: editorconfig info extraction

### PR DESCRIPTION
**Pull request by satwikkansal**

### CODE REVIEW

**Comment by satwikkansal** [comment]
Please review top 2 commits. Previous commits are based on #102

File: coala_quickstart/info_extraction/Information.py [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+
+class FinalNewlineInfo(StyleInfo):
+    description = ("Information representing whether to enformce file ending "
+                   " with a newline or not.")
+    value_type = (bool,)
+
+
+class CharsetInfo(StyleInfo):
+    description = "Ifnormation regarding character set."
```
Comment (adtac) [inline coala_quickstart/info_extraction/Information.py]: typo

File: coala_quickstart/info_extraction/Information.py [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+    value_type = (str,)
+
+
+class LineBreaksInfo(StyleInfo):
+    description = ('Infromration regarding how line breaks are represented '
```
Comment (adtac) [inline coala_quickstart/info_extraction/Information.py]: typo

**Comment by adtac** [comment]
ack https://github.com/coala/coala-quickstart/pull/122/commits/f7aa29be23d6ba94d7ba5b54f782bc56cb756833 https://github.com/coala/coala-quickstart/pull/122/commits/78bbfb2e893c02f35493f85660dc13dd4511bc8f

**Comment by satwikkansal** [comment]
reack a1f9dc0 37573da

**Comment by adtac** [comment]
@rultor merge

**Pull request merged**